src/stacker/ConcreteStacker.ts

```diff
@@ -115,6 +134,27 @@ export class ConcreteStacker implements Stacker {
   private async updatePRDescriptionsForCompleteTreeContainingCommit(
     commit: Commit,
   ): Promise<void> {
-    // TODO: Implement
+    const stack = [];
+    const nextCommits = [commit];
+    while (nextCommits.length) {
+      const nextCommit = nextCommits.pop()!;
+      stack.push(nextCommit);
+      const childCommits: Commit[] = [];
+      nextCommit.childCommits.forEach((commitHash) => {
+        this.sourceControl.getCommitByHash(commitHash);
+      });
+      nextCommits.push(...childCommits);
+    }
+    //TODO: Implement a complete version of the stack that start from the merge-base commit and also takes into consideration landed PRs
+    let commitPrInfoPairs: { commit?: Commit; prInfo: PullRequestInfo }[] = [];
```

Comment:
We can actually use `const` here since we're not reassigning `commitPrInfoPairs`

```suggestion
    const commitPrInfoPairs: { commit?: Commit; prInfo: PullRequestInfo }[] = [];
```